packages/governance/src/contractGovernor.js

```diff
@@ -151,7 +151,7 @@ harden(validateQuestionFromCounter);
  *   creatorFacet: GovernorCreatorFacet<SF>,
  *   publicFacet: GovernorPublic,
  * }>}
- * @param {import('@agoric/vat-data').Baggage} baggage
+ * @param {import('@agoric/swingset-liveslots').Baggage} baggage
```

Comment:
I don't see `Baggage` in the generated docs for swingset-liveslots. I wonder what's up here.